src/analyses/apron/apronAnalysis.apron.ml

```diff
@@ -19,6 +20,10 @@ struct
 
   open AD
 
+  open ApronPrecCompareUtil.Util
```

Comment:
This `Util.Util` is awkward. Why not just include its contents directly at the top level of the module file?

Reply:
Changed it, so the `.Util` is no longer needed.